core/app/finders/spree/posts/find.rb

```
@@ -0,0 +1,136 @@
+    class Find < Spree::BaseFinder
+      def initialize(scope:, params:)
+        @scope = scope
+        @query = params[:q].presence&.strip
+        @ids = String(params.dig(:filter, :ids)).split(',')
+        @slugs = String(params.dig(:filter, :slugs)).split(',')
+        @category_ids = String(params.dig(:filter, :category_ids)).split(',')
+        @author_ids = String(params.dig(:filter, :author_ids)).split(',')
+        @tags = params.dig(:filter, :tags).to_s.split(',').compact_blank
+        @sort_by = params[:sort_by]
+        @published = params.dig(:filter, :published)
+      end
```

RuboCop (1.76.1)

[warning] Call `super` to initialize state of the parent class.

(Lint/MissingSuper)